forth_src/dos.fs

```diff
@@ -0,0 +1,14 @@
+require io
+
+\ send command string to drive and
+\ print response
+: send-cmd ( addr len -- )
+0 0 15 15 open 
+15 chkout type \ send command
+clrch 15 chkin
+refill source type cr \ print result
+clrch refill 15 close ;
```

Comment:
The first `refill` reads the drive status into the input buffer, `source type` prints it but doesn't consume the buffer, so the second `refill` is needed in order to flush the buffer and prevent it being interpreted as input.

 It's kind of a hack, yes, but it seemed to be the quickest and easiest way to do it with the existing wordset.

Reply:
Ah! That seems like the right way to do it!

Reply:
Thinking about it further, it doesn't really work. Imagine following case:

    : fmt s" N0:NAME,ID" send-cmd ;
    fmt fmt fmt fmt fmt

This would only format the drive once... right?

Reply:
I came to think of a nice can of worms now. What if this code runs while a file is being INCLUDED?

Reply:
> 
> 
> I came to think of a nice can of worms now. What if this code runs while a file is being INCLUDED? I suspect the new words would not restore things nicely. Same with the already existing words in disk.asm (except INCLUDED itself).
> 
> Food for thought :D No action required.

Yeah, there's definitely potential for Bad Things to happen if immediate-mode code tries to use a file# that INCLUDE already has open, or doesn't clean up after itself, but I think that it should be safe if used carefully.